cmd/main.go

```diff
@@ -145,6 +147,13 @@ func main() {
 
 	var openshiftVersionsMap models.OpenshiftVersions
 
+	if !isJSONString(Options.OpenshiftVersions) {
```

Comment:
maybe it will be better to add method: initOCPDefaultVersions and handle everything inside this method